resdk/query.py

```diff
@@ -280,7 +280,10 @@ def get(self, *args, **kwargs):
                 raise ValueError('Non-keyworded arguments cannot be combined with keyworded ones.')
 
             arg = args[0]
-            kwargs = {'id': arg} if str(arg).isdigit() else {self.slug_field: arg}
+            if str(arg).isdigit():
+                kwargs = {'id': arg}
+            else:
+                kwargs = {self.slug_field: arg, 'ordering': '-version', 'limit': 1}
```

Comment:
This does not behave the same if `slug` is passed as a keyword or positional argument.